build.gradle.kts

```diff
@@ -20,7 +20,7 @@ group = "org.example"
 version = "1.0-SNAPSHOT"
 description = "embedding"
 
-var graalVMVersion: String = "23.1.2";
+var graalVMVersion: String = "24.0.1";
```

Comment:
Don't we rather use the LTS release 23.1.3?

Reply:
It depends on what we want to test. I think either is fine. We have tests elsewhere for backward compatibility.